Panto/css/typography.css

```diff
@@ -0,0 +1,36 @@
+:root {
+  /* font family */
+  --font-family-text: "Open Sans", sans-serif;
+
+  /* font size */
+  --font-size-html: 62.5%;
+  --font-size-panto-logo-type: 2.8rem;
+
+  --font-size-h1: 8rem;
+  --font-size-h2: 4.2rem;
+  --font-size-h3: 2.4rem;
+  --font-size-subheading: 1.8rem;
+
+  --font-size-main-nav: 1.8rem;
+  --font-size-shopping-cart-icon: 2.4rem;
+
+  --font-size-more-info: 1.4rem;
+  --font-size-view-all: 1.8rem;
+
+  --font-size-footer-heading: 1.8rem;
```

Comment:
have you noticed that you have lots of repetitive values? like 1.4rem, 1.8rem, 2.4rem, and so on.
This is because these values are selected from a design system. It is better to first define these values as css properties like:
```css
--font-size-1: 0.2rem;
--font-size-2: 0.4rem;
--font-size-3: 0.8rem;
--font-size-4: 1.4rem;
--font-size-5: 1.8rem;
/* and so on... */
```

the design file you are working on doesn't have a design system, so you have to create yourself a list of font sizes similar to the one I created. (don't overthink it, the implementation doesn't have to go pixel for pixel of the design file)